Makefile.pre.in

```diff
@@ -1419,6 +1419,8 @@ libinstall:	build_all $(srcdir)/Modules/xxmodule.c
 			echo $(INSTALL_DATA) $$i $(LIBDEST); \
 		fi; \
 	done
+	@# bpo-37468: Don't install distutils/command/wininst-*.exe files used
+	@# by distutils bdist_wininst: bdist_wininst only works on Windows.
```

Comment:
Should this comment be above line 1436? This for loop does other stuff, right?

Reply:
I wasn't sure if it is possible to put comments into this loop which uses \ at end of lines for multiline expression. It looks like it works, so I moved the comment inside the loop.